Dockerfile_mbtci_template

```diff
@@ -12,9 +12,19 @@ ARG SAPMACHINE_VERSION=JAVA_VERSION_TEMPLATE
 
 # Environment variables
 ENV PYTHON /usr/bin/python3
+
 ENV MAVEN_HOME /usr/share/maven
 ENV M2_HOME ${MAVEN_HOME}
-ENV PATH /usr/local/go/bin:$PATH
+ENV JAVA_HOME /opt/jdk
+ENV PATH $PATH:$MAVEN_HOME/bin:${JAVA_HOME}/bin
+
+ENV NODE_HOME /usr/local/nodejs
+ENV PATH $PATH:${NODE_HOME}/bin
+
+ENV GOPATH /go
```

Comment:
No needed, it duplicates GO_HOME purpose

Reply:
GOPATH env already exist in the past (https://github.com/SAP/cloud-mta-build-tool/commit/35393669a1319333777bf26c864976307a496f0b#diff-3632a131a98f4a6db44b6238249146e491fa94be0b683a407b11498c3bffc11c)

when we merge all docker files into Dockerfile_mbtci_template, GOPATH is deleted, now I just revert it.

As to GO_HOME env, I will revert it, and when install go, use GO_HOME as a local var

Reply:
It was not needed at that time also. JAVA_HOME is the only env variable needed (defined by the java standard). 

The goal is to keep thing simple and straightforward, so adding paths indirection in the image via symlinks or env vars only and only if strictly needed, the filesystem hierarchy standard is already fulfilled in the source archive. 
With all due respect, the original Dockerfile does not have anything in it that shall be considered as material for its rewrite. 

And you should consider not redoing PR already done and working as a maintainer. You just have to comment and wait for contributors to take comments into account and run the CI validation on PR, like for example the maintainer has done here: https://github.com/carlossg/docker-maven/pull/304. And then merge. 
The maintainer workload is already enough without adding to it unneeded PR redo extra work.